docs/proposals/cri.md

```diff
@@ -0,0 +1,308 @@
+---
```

Comment:
We already have a proposal PR. Please delete this file from here

Reply:
Done